src/WebJobs.Extensions.DurableTask.Analyzers/SyntaxNodeUtils.cs

```diff
@@ -29,16 +29,34 @@ public static DurableVersion GetDurableVersion(SemanticModel semanticModel)
 
         public static bool TryGetSemanticModelForSyntaxTree(SemanticModel model, SyntaxNode node, out SemanticModel newModel)
         {
-            if (model?.SyntaxTree == null || node?.SyntaxTree == null)
+            try
+            {
+                if (model?.SyntaxTree == null || node?.SyntaxTree == null)
```

Comment:
nit: this try catch can be moved outside the try-catch.

In general, if you have identified which method can throw the ArgumentException, I would only wrap that code in the try-catch to make it more clear where we expect the exception to be thrown.